resolwe/flow/models.py

```diff
@@ -3,12 +3,24 @@
 Flow Models
 ===========
 
+Base Model
+**********
```

Comment:
It's strange to see stars *** a header level below equality ===. I prefer not to use stars at all:
```
===========
Flow Models
===========

Base Model
==========
```